rollup.config.js

```diff
@@ -1,17 +1,47 @@
+import { nodeResolve } from "@rollup/plugin-node-resolve";
 import commonjs from "@rollup/plugin-commonjs";
+import replace from "@rollup/plugin-replace";
 import { terser } from "rollup-plugin-terser";
 
-function createConfig(format) {
+function createConfig(format, options = {}) {
   const dir = format === "module" ? "esm" : format;
+  const { resolved = false, min = false } = options;
   return {
-    input: require.resolve("autopublish-template"),
+    input: require.resolve("react-emotion/dist/index.esm"),
     output: {
-      file: `${dir}/index.js`,
+      file: `${dir}/index${resolved ? ".resolved" : ""}${min ? ".min" : ""}.js`,
       sourcemap: true,
       format,
+      paths: {
+        react: resolved
+          ? `//cdn.jsdelivr.net/npm/@esm-bundle/react@16.13.1/esm/react.production.min.js`
```

Comment:
Instead of hard coding this, I think it would be better to fetch the latest version of react and use that one. Like how react-dom does it.

Reply:
This was marked at resolved, but not updated. See https://github.com/esm-bundle/react-dom/blob/65d2c766dcc2ab18c341eeb3a089d79abbe3666b/rollup.config.js#L8

Reply:
Oops, yea I updated only one place. Updated. I believe this doesn't need to use a specific version of react like that react-dom example you linked to. This package is dependent on a specific version of emotion but not react. I could be wrong though.

Reply:
Yea that method wouldn't work for this package anyway because it doesn't have a version tied to it in package.json 
https://github.com/esm-bundle/react-emotion/blob/429834cfb6c4bcfeea05f4e850df571a5f31a1ce/package.json#L51

Reply:
If anybody is using this in production, it's not good to link to the latest version of react/react-dom. The user will want to only use one version of react across all their shared libs. I'd recommend pinning to a specific version of react in the package.json.

This area is where esm-bundle isn't the best -- import maps are truly the solution for version management. But without import maps, pinning to a specific version is best.